server/transaction_test.go

```diff
@@ -256,7 +256,7 @@ func Test_Transaction_InvalidIDGiven(t *testing.T) {
 			t.Fatal("expected", http.StatusInternalServerError, "got", w.Code)
 		}
 
-		if !strings.Contains(w.Body.String(), "invalid transaction ID: does not match") {
+		if !strings.Contains(w.Body.String(), "does not match") {
```

Comment:
I aligned the weird string match here with the "new reality". Somehow the message creation within the error stacks changed slightly. Uncertain why and if this has bigger implications. I just moved on with this.